Lib/unittest/mock.py

```diff
@@ -1021,15 +1016,14 @@ def _mock_check_sig(self, *args, **kwargs):
         pass
 
 
-    def __call__(_mock_self, *args, **kwargs):
+    def __call__(self, /, *args, **kwargs):
```

Comment:
mock recently added `AsyncMock` whose helpers and `async def __call__` can also be refactored like the synchronous API.